icicle-repl/test/cli/repl/t30.3-sum-not-error/expected

```diff
@@ -41,20 +41,20 @@ typedef struct {
 
   /* compute for (0,0) */
     /* outputs */
-    ibool_t          repl_ix_0;
-    iint_t           repl_ix_1;
-    idouble_t        repl_ix_2;
+    ibool_t          repl_24ix_240;
+    iint_t           repl_24ix_241;
+    idouble_t        repl_24ix_242;
 
     /* resumables: values */
-    idouble_t        res_0_0_acc_perhaps_conv_5_simpflat_6;
-    iint_t           res_0_0_acc_perhaps_conv_5_simpflat_5;
-    ibool_t          res_0_0_acc_perhaps_conv_5_simpflat_4;
+    idouble_t        res_0_0_acc_24perhaps_24conv_245_24simpflat_246;
+    ibool_t          res_0_0_acc_24perhaps_24conv_245_24simpflat_244;
+    iint_t           res_0_0_acc_24perhaps_24conv_245_24simpflat_245;
```

Comment:
it's a shame these become a bit harder to read, apparently `$` is actually accepted by `gcc` and `clang`, maybe we should pass it through? :trollface: